src/app/common/NavTabs.vue

```diff
@@ -2,7 +2,7 @@
   <KTabs
     :tabs="kTabs"
     :model-value="currentTabHash"
-    :show-panels="false"
+    hide-panels
```

Comment:
**Comment**: Noticed we were using the wrong prop (which wasn’t even called `showPanels`). Not sure if that changed earlier already.